### ISSUE TITLE
Will not import a newly recorded MP3 but will import other MP3s

### ISSUE THREAD

**Issue by JimFlasch** [SartoxSoftware/audacium#98]
**Describe the bug**
Will not import a newly recorded MP3 but will import other MP3s

**To Reproduce**
1. use a an MP3 converter to convert a cassette tape to MP3.
2. attempt to import newly recorded MP3 into Audacium and get message saying "Audacity did not recognize the type of file '...*.mp3. Try installing FFmpeg" even if the proper library is installed.
3. attempt to import another mp3 file that's already been on your hard drive for awhile.
4. see it import just fine with no error message

**Expected behavior**
BOTH mp3s should import without problems!

**Additional information (please complete the following information):**
 - OS: Windows 10
 - Version Audacium 1.0.0-beta-20211210 (according to the about screen, even after installing version 1.0.0 Release Build.)

**Comment by AnErrupTion** [comment]
I'm guessing the converter has a problem. What's the name of it?

**Comment by JimFlasch** [comment]
Blumway cassette tape to mp3 converter. And this is the first time Audacium / Audacity has had a problem importing the mp3s it creates - and only the mp3s it creates. It's handled them fine before.

**Comment by JimFlasch** [comment]
And suddenly, without my doing anything to cause it, Audacium can't find my recording and playback devices!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WHY????????????????????????????????

**Comment by AnErrupTion** [comment]
@JimFlasch Can you, when you launch Audacium, go to the Help menu on the top and show the logs? There should be a section called "Logs" and in there you should be able to show a Log window, where you basically get all the logs. Perhaps there'll be more information.

**Comment by AnErrupTion** [comment]
Right. You need to download a specific version of FFmpeg for Audacium (5.0 is too recent). Go there to see how to install it: https://manual.audacityteam.org/man/installing_ffmpeg_for_windows.html. Then, manually locate FFmpeg again (go to the settingts, then Libraries, then Find FFmpeg or something like that)

**Comment by JimFlasch** [comment]
Alright, now I can import the  file I couldn't import before; however, my other, more recent problem - the sudden inability to find  and recognize my recording and playback devices (as of 5 days ago) - persists. How do I fix that?

**Comment by AnErrupTion** [comment]
@JimFlasch Could you send your logs now please?

**Comment by JimFlasch** [comment]
Here's my latest log:

23:01:44: Audacium 1.0.0-beta-20211210
23:01:44: Trying to load FFmpeg libraries...
23:01:44: mLibAVFormatPath ('C:\FFMPEG\ffmpeg-5.0-essentials_build\bin\avformat-55.dll') is not empty. Loading from it.
23:01:44: Looking up PATH environment variable...
23:01:44: PATH = 'C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Calibre2\;C:\Program Files (x86)\Common Files\Acronis\SnapAPI\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile64\;C:\Program Files (x86)\Common Files\Acronis\FileProtector\;C:\Program Files (x86)\Common Files\Acronis\FileProtector64\;C:\Users\James\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Bandizip\'
23:01:44: Temporarily prepending ';C:\FFMPEG\ffmpeg-5.0-essentials_build\bin' to PATH...
23:01:44: Checking for monolithic avformat from 'C:\FFMPEG\ffmpeg-5.0-essentials_build\bin\avformat-55.dll'.
23:01:45: Error: Failed to load shared library 'C:\FFMPEG\ffmpeg-5.0-essentials_build\bin\avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
23:01:45: Loading avutil from ''.
23:01:45: Error: Failed to load shared library '.dll' (error 126: The specified module could not be found.)
23:01:45: Loading avcodec from ''.
23:01:45: Error: Failed to load shared library '.dll' (error 126: The specified module could not be found.)
23:01:45: Loading avformat from 'C:\FFMPEG\ffmpeg-5.0-essentials_build\bin\avformat-55.dll'.
23:01:45: Error: Failed to load shared library 'C:\FFMPEG\ffmpeg-5.0-essentials_build\bin\avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
23:01:45: Error: Failed to load FFmpeg libraries.
23:01:45: Returning PATH to previous setting: C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Calibre2\;C:\Program Files (x86)\Common Files\Acronis\SnapAPI\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile64\;C:\Program Files (x86)\Common Files\Acronis\FileProtector\;C:\Program Files (x86)\Common Files\Acronis\FileProtector64\;C:\Users\James\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Bandizip\
23:01:45: Trying to load FFmpeg libraries from default path, 'C:\Program Files\FFmpeg For Audacity\avformat-55.dll'.
23:01:45: Looking up PATH environment variable...
23:01:45: PATH = 'C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Calibre2\;C:\Program Files (x86)\Common Files\Acronis\SnapAPI\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile64\;C:\Program Files (x86)\Common Files\Acronis\FileProtector\;C:\Program Files (x86)\Common Files\Acronis\FileProtector64\;C:\Users\James\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Bandizip\'
23:01:45: Temporarily prepending ';C:\Program Files\FFmpeg For Audacity' to PATH...
23:01:45: Checking for monolithic avformat from 'C:\Program Files\FFmpeg For Audacity\avformat-55.dll'.
23:01:45: Error: Couldn't find symbol 'avutil_version' in a dynamic library (error 127: The specified procedure could not be found.)
23:01:45: Error: Couldn't find symbol 'avcodec_version' in a dynamic library (error 127: The specified procedure could not be found.)
23:01:45: avformat not monolithic
23:01:45: Loading avutil from 'avutil-52.dll'.
23:01:45: Loading avcodec from 'avcodec-55.dll'.
23:01:45: Loading avformat from 'C:\Program Files\FFmpeg For Audacity\avformat-55.dll'.
23:01:45: Actual avutil path C:\Program Files\FFmpeg For Audacity\avutil-52.dll
23:01:45: Actual avcodec path C:\Program Files\FFmpeg For Audacity\avcodec-55.dll
23:01:45: Actual avformat path C:\Program Files\FFmpeg For Audacity\avformat-55.dll
23:01:45: Importing symbols...
23:01:45: All symbols loaded successfully. Initializing the library.
23:01:45: Retrieving FFmpeg library version numbers:
23:01:45:    AVCodec version 0x373466 - 55.52.102 (built against 0x373466 - 55.52.102)
23:01:45:    AVFormat version 0x372164 - 55.33.100 (built against 0x372164 - 55.33.100)
23:01:45:    AVUtil version 0x344264 - 52.66.100 (built against 0x344264 - 52.66.100)
23:01:45: Returning PATH to previous setting: C:\Program Files\Common Files\Oracle\Java\javapath;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Calibre2\;C:\Program Files (x86)\Common Files\Acronis\SnapAPI\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile\;C:\Program Files (x86)\Common Files\Acronis\VirtualFile64\;C:\Program Files (x86)\Common Files\Acronis\FileProtector\;C:\Program Files (x86)\Common Files\Acronis\FileProtector64\;C:\Users\James\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Bandizip\
23:01:45: FFmpeg libraries loaded successfully.


Like I said in my last post, I can now import newly recorded mp3 files; it's the sudden inability of Audacium to find and recognize my recording and playback devices (as of 22 days ago, now) that's my current problem.
